src/sql/parts/connection/connectionDialog/recentConnectionTreeController.ts

```diff
@@ -126,17 +126,19 @@ export class RecentConnectionTreeController extends DefaultController {
 		}
 
 		let anchor = { x: event.x + 1, y: event.y };
-		this._contextMenuService.showContextMenu({
-			getAnchor: () => anchor,
-			getActions: () => this.actionProvider.getActions(tree, element),
-			onHide: (wasCancelled?: boolean) => {
-				if (wasCancelled) {
-					tree.DOMFocus();
-				}
-			},
-			getActionsContext: () => (actionContext)
-		});
-
-		return true;
+		if (anchor.x && anchor.y) {
```

Comment:
why is this getting called with an event without an x,y pos?  it seems to be invoked twice, is there some underlying bug we should fix in how the command is invoked?  also, is the correct condition to determine the duplicate events really the x,y fields, or is there are more generic condition to check?  also, please add a comment to the code why the check is needed.